packages/apps/plugins/plugin-layout/src/components/Toast.tsx

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React from 'react';
+
+import type { Toast as ToastProps } from '@dxos/app-framework';
+import { Button, Toast as NaturalToast } from '@dxos/react-ui';
+
+export const Toast = ({
+  title,
+  description,
+  icon,
+  duration,
+  actionLabel,
+  actionAlt,
+  closeLabel,
+  onAction,
+}: ToastProps) => {
+  return (
+    <NaturalToast.Root defaultOpen duration={duration}>
```

Comment:
I was wondering if the `Natural{Component}` naming would stick around 😅

Reply:
It's not my favourite, but I don't have a better alternative currently 😅